doc/source/contributing.rst

```diff
@@ -5,22 +5,853 @@
 Contributing to MDAnalysis
 ===========================
 
-Please report **bugs** or **enhancement requests** through the `Issue
-Tracker`_. Questions can also be asked on the `mdnalysis-discussion mailing
-list`_.
+MDAnalysis is a free and open source project. It evolves and grows with the demands of its user base. 
+The development team very much welcomes contributions from its users. 
+Contributions can take many forms, such as:
+
+    * **bug reports** or **enhancement requests** filed through the `Issue Tracker`_
+    * **source code** for fixing bugs or new functionality (e.g. new analysis tasks)
+    * **documentation** (either in the code or on the user guide)
+    * **questions** or **discussions** on the `mdnalysis-discussion`_ mailing list
 
 The MDAnalysis community subscribes to a `Code of Conduct`_ that all community
 members agree and adhere to --- please read it.
 
-.. _Issue Tracker: https://github.com/MDAnalysis/mdanalysis/issues
-.. _`mdnalysis-discussion mailing list`:
-   http://groups.google.com/group/mdnalysis-discussion
-.. _`Code of Conduct`: https://www.mdanalysis.org/pages/conduct/
+.. note::
+
+    Parts of this page came from the `Contributing to pandas <http://pandas.pydata.org/pandas-docs/stable/contributing.html>`_ guide.
+
+Where to start?
+===============
+
+All contributions, bug reports, bug fixes, documentation improvements,
+enhancements, and ideas are welcome.
```

Comment:
I think somewhere in this page there needs to be a quick paragraph explaining the license used (GPL) and that contributions made will need to fall under this.

Reply:
Thanks for the review! What do you think of [this](https://lilyminium.github.io/UserGuide/contributing.html)?